### ISSUE TITLE
Send templates based on Mailable class

### ISSUE THREAD

**Issue by joostdebruijn** [coconutcraig/laravel-postmark#125]
## Detailed description

It would be nice to be able to send mails with a template based on the Mailable class. Now it is only possible to send mails based on templates via Notifications.

## Context

I don't want to use the more advanced Notifications-stuff, but I would like to use the Mail functionality with Postmark templates.

## Possible implementation

It would be nice if a template and data could be provided in `build()` of a Mailable.

## Your environment

Include as many relevant details about the environment you experienced the bug in and how to reproduce it.

- Laravel Postmark Version: 2.10.0
- Laravel Version: 8.45.0
- PHP Version: 8.0.7

**Comment by joostdebruijn** [comment]
I have implemented this way now, works quite nice:

```php
<?php

namespace App\Mail;

use Illuminate\Bus\Queueable;
use Illuminate\Contracts\Queue\ShouldQueue;
use Illuminate\Mail\Mailable;
use Illuminate\Queue\SerializesModels;

class PostmarkTemplate extends Mailable
{
    use Queueable, SerializesModels;

    public $alias;
    public $model;

    /**
     * Create a new message instance.
     *
     * @return void
     */
    public function __construct($alias, $data)
    {
        $this->alias = $alias;
        $this->model = $data;
    }

    /**
     * Build the message.
     *
     * @return $this
     */
    public function build()
    {
        return $this
            ->view('postmark::template');
    }
}
```

**Comment by craigpaul** [comment]
Hey @joostdebruijn, I think this could be quite useful. I don't have the time to implement this myself right now, but if you wanna submit a PR to get this moving in the meantime, that would be awesome 👍

**Comment by craigpaul** [comment]
Fixed via #130
